WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIV0.swift

```diff
@@ -0,0 +1,90 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class ConversationsAPIV0: ConversationsAPI, VersionedAPI {
+
+    // MARK: - Constants
+
+    enum Constant {
+        static let batchSize = 500
+    }
+
+    // MARK: - Properties
+
+    var apiVersion: APIVersion { .v0 }
+
+    let httpClient: HTTPClient
+
+    // MARK: - Initialize
+
+    init(httpClient: HTTPClient) {
+        self.httpClient = httpClient
+    }
+
+    func getLegacyConversationIdentifiers() async throws -> PayloadPager<UUID> {
```

Comment:
```suggestion
    public func getLegacyConversationIdentifiers() async throws -> PayloadPager<UUID> {
```
isn't it missing the visibility?

Reply:
good point, but I think it doesn't need to be `public` because the whole `ConversationsAPI0` isn't. the outer caller just uses the `ConversationsAPIBuilder` and sees the protocol.